Essentials/src/com/earth2me/essentials/commands/Commanditemlore.java

```diff
@@ -0,0 +1,76 @@
+package com.earth2me.essentials.commands;
+
+import com.earth2me.essentials.User;
+import com.earth2me.essentials.utils.FormatUtil;
+import com.earth2me.essentials.utils.NumberUtil;
+import com.google.common.collect.Lists;
+import org.bukkit.Server;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.ItemMeta;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import static com.earth2me.essentials.I18n.tl;
+
+public class Commanditemlore extends EssentialsCommand {
+
+    public Commanditemlore() {
+        super("itemlore");
+    }
+
+    @Override
+    protected void run(Server server, User user, String commandLabel, String[] args) throws Exception {
+        ItemStack item = user.getBase().getItemInHand();
+        if (item.getType().name().contains("AIR")) {
+            user.sendMessage(tl("itemloreInvalidItem"));
+            return;
```

Comment:
```suggestion
            throw new Exception(tl("itemloreInvalidItem"));
```
This is probably better, to remain consistent with the way you are handling errors in other parts of this command.